Makefile

```diff
@@ -6,12 +6,21 @@ OBJ := start.o \
 
 CFLAGS := -O0 -ffreestanding -nostdlib -Wall -Wextra -g
 
+# This lets us override CC and LD with environment variables
```

Comment:
Aren't `CC` and `LD` already environment variables? If they're set in the shell session before invoking `make` won't the correct ones be invoked?

Reply:
You're right, thanks